.github/workflows/buildTestBinaries.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build test binaries
+
+# Controls when the action will run. Triggers the workflow push is only on main branch and PR on any branch.
+on:
+  pull_request:
```

Comment:
What about main branch builds? Don't we want to publish also latest builds?

Reply:
No. This is just for the PR for master it must be separate job (where you want to store builds at artifacts ? )

Reply:
Same as for the PR. In the job itself.
I would access it by clicking on the pipeline success flag and get to it.
![image](https://user-images.githubusercontent.com/6858585/176445382-54cb9ad0-6d50-4381-b8b3-ba10f3634516.png)

Reply:
In my opinion it's ok for latest main builds. People should anyway use latest release, and that's visible on the right place.

Reply:
![image](https://user-images.githubusercontent.com/23374974/176447126-a3d3b6e2-e4c6-43ab-884a-0699f78cc88a.png)

Reply:
added for `main` branch